src/Output/StrictUnifiedDiffOutputBuilder.php

```diff
@@ -0,0 +1,325 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of sebastian/diff.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SebastianBergmann\Diff\Output;
+
+use SebastianBergmann\Diff\ConfigurationException;
+
+/**
+ * Strict Unified diff output builder.
+ *
+ * @name Unified diff output builder
+ *
+ * @description Generates (strict) Unified diff's (unidiffs) with hunks.
+ *
+ * @author SpacePossum
+ *
+ * @api
+ */
+final class StrictUnifiedDiffOutputBuilder implements DiffOutputBuilderInterface
+{
+    /**
+     * @var int
+     */
+    private static $noNewlineAtOEFid = 998877;
```

Comment:
what is this value ?

Reply:
https://github.com/sebastianbergmann/diff/blob/master/src/Differ.php#L149

the differ injects entries as follows:
- 2 /* REMOVED */
- 1 /* ADDED */
- 0 /* OLD */
- 3 /* custom warning of diff (i.e. _not_ udiff) */

I needed a new entry for the EOF warning that is defined in the udiff format;
for that I picked this number

Reply:
so it could be modified?
value is still weird, why not 4 or 9? no clue.
it's supper different :D

Also, same value/property is duplicated in `UnifiedDiffOutputBuilder`.

maybe those values shall be extracted to ~enum class (that is internal) and reused?